app/views/users/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="flex flex-col gap-4 justify-center items-center w-full h-max">
+  <p style="color: green"><%= notice %></p>
+  <h1 class="text-sky-600/80 font-bold text-3xl"> Budgetty</h1>
+  <%= image_tag ('finance-report-research.png'), class:"w-96 h-84"%>
+  <div class="flex gap-8 mt-8">
+    <%= link_to "Log in", new_user_session_path,  class: "bg-sky-600/80 p-2 border border-cyan-300  rounded  " %>
+    <%= link_to "Sign up", new_user_registration_path ,  class: "bg-sky-600/80 p-2 border border-cyan-300  rounded  "%>
+    <%= link_to "Categories", categories_path ,  class: "bg-sky-600/80 p-2 border border-cyan-300  rounded  "%>
+  </div>
+  <div class="flex">
+  <a class="font-light" href="https://iconscout.com/3ds/finance-report-research" target="_blank">Finance Report Research 3D Illustration</a> &nbsp; by &nbsp;<a href="https://iconscout.com/contributors/iqonic-design">iqonic.design</a> &nbsp; on &nbsp;<a href="https://iconscout.com">IconScout</a>
+  </div>
+</div>
```

Comment:
- [ ] After the user login, the page ı have to see the categories page, not this page in the picture.
[Requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=When%20the%20user%20logs%20in%2C%20they%20are%20presented%20with%20the%20categories%20page.)
![image](https://user-images.githubusercontent.com/38635158/167253947-d15044dd-a72f-421b-a096-8b40b637df45.png)